services/workflows-service/src/workflow/hook-callback-handler.service.ts

```diff
@@ -44,13 +43,23 @@ export class HookCallbackHandlerService {
       );
     }
 
+    if (processName === 'website-monitoring') {
+      return await this.prepareWebsiteMonitoringContext(
+        data,
+        workflowRuntime,
+        resultDestinationPath,
+        currentProjectId,
+      );
+    }
+
     const removeLastKeyFromPath = (path: string) => {
       return path?.split('.')?.slice(0, -1)?.join('.');
     };
 
     const resultDestinationPathWithoutLastKey = removeLastKeyFromPath(resultDestinationPath);
     const result = get(workflowRuntime.context, resultDestinationPathWithoutLastKey);
 
+    //@ts-ignore
```

Comment:
Why is this needed?

Reply:
P3

Reply:
Because property `status` does not exist on type `object`

Reply:
`isObject` is imported from `lodash` instead of from `@ballerine/common`.